src/main/java/org/folio/rest/impl/BatchVoucherExportConfigHelper.java

```diff
@@ -111,4 +114,23 @@
     return handleGetRequest(endpoint, httpClient, ctx, okapiHeaders, logger)
       .thenCompose(json -> VertxCompletableFuture.supplyBlockingAsync(ctx, () -> json.mapTo(ExportConfigCollection.class)));
   }
+
+  public CompletableFuture<String> testUploadUri(String id) {
+    CompletableFuture<ExportConfig> exportConfigFut = getExportConfig(id);
```

Comment:
Two endpoints from the mod-invoice-storage are called here. However, the necessary permissions for this API have not been added. However, the necessary modulePermissions for `/batch-voucher/export-configurations/{id}/credentials/test` API have been missed.